Tests/FunctionalTests/TraitTests.swift

```diff
@@ -560,24 +560,28 @@ struct TraitTests {
         buildSystem: BuildSystemProvider.Kind,
         configuration: BuildConfiguration,
     ) async throws {
-        try await fixture(name: "Traits") { fixturePath in
-            let (stdout, _) = try await executeSwiftPackage(
-                fixturePath.appending("Package10"),
-                configuration: configuration,
-                extraArgs: ["dump-symbol-graph", "--experimental-prune-unused-dependencies"],
-                buildSystem: buildSystem,
-            )
-            let optionalPath = stdout
-                .lazy
-                .split(whereSeparator: \.isNewline)
-                .first { String($0).hasPrefix("Files written to ") }?
-                .dropFirst(17)
+        try await withKnownIssue(isIntermittent: true, {
```

Comment:
question: The test was previously passing on Windows, but we now marked it as `intermittent`. do we understand why?